docker/build-tools/Dockerfile.centos

```diff
@@ -28,10 +28,9 @@ RUN echo "/opt/rh/httpd24/root/usr/lib64" > /etc/ld.so.conf.d/httpd24.conf && \
 # We need gsutil to push/pull
 ADD https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-265.0.0-linux-x86_64.tar.gz /tmp
 RUN tar -xzvf "/tmp/google-cloud-sdk-265.0.0-linux-x86_64.tar.gz" -C /usr/local
-
-ENV LLVM_RELEASE=clang+llvm-10.0.0-x86_64-linux-centos7
+ENV LLVM_RELEASE=clang-toolchain_7f855ee4c72c288ff06e0b29bdeab23ddcb022a0_clang+llvm-12.0.1-x86_64-linux-centos7.tar
 # hadolint ignore=DL4006
-RUN curl -fsSL --output ${LLVM_RELEASE}.tar.xz http://storage.googleapis.com/getenvoy-package/clang-toolchain/a79bbfea57c05b5cd2708ab432192be0e187e074/${LLVM_RELEASE}.tar.xz && \
+RUN curl -fsSL --output ${LLVM_RELEASE}.tar.xz http://storage.googleapis.com/istio-build-deps/${LLVM_RELEASE}.tar.xz && \
```

Comment:
nit: would be nice to just comment that the instructions for this image are in <link to readme>